packages/ui/src/ERC1155Controls.svelte

```diff
@@ -13,6 +13,11 @@
     ...erc1155.defaults,
     info: { ...infoDefaults }, // create new object since Info is nested
   };
+
+  let requireAccessControl: boolean;
+  $: {
+    requireAccessControl = erc1155.isAccessControlRequired(opts);
+  }
```

Comment:
Shorthand:

```suggestion
  $: requireAccessControl = erc1155.isAccessControlRequired(opts);
```

[Docs](https://svelte.dev/docs#component-format-script-3-$-marks-a-statement-as-reactive)